content/v2.0/influxdb-templates/use.md

```diff
@@ -1,20 +1,193 @@
+
+```sh
+# Syntax
+influx pkg -f <template-file-path>
+
+# Examples
+# Install a single template
+influx pkg -f /path/to/template.yml
+
+# Install multiple templates
+influx pkg \
+  -f /path/to/this/template.yml \
+  -f /path/to/that/template.yml
+```
+
+#### Install all templates in a directory
+To install all templates in a directory, use the `-f` or `--file` flag to provide
+the **directory path** of the directory where template manifests are stored.
+By default, this only installs templates stored in the specified directory.
+To install all templates stored in the specified directory and its subdirectories,
+include the `--recurse` flag.
+
+```sh
+# Syntax
+influx pkg -f <template-directory-path>
+
+# Examples
+# Install all templates in a directory
+influx pkg -f /path/to/template/dir/
+
+# Install all templates in a directory and its subdirectories
+influx pkg -f /path/to/template/dir/ --recurse
+```
+
+### Install templates from URLs
+To install templates from a URL, use the `-u` or `--url` flag to provide the URL
+of the template manifest.
+
+```sh
+# Syntax
+influx pkg -u <template-url>
+
+# Examples
+# Install a single template from a URL
+influx pkg -u https://mydomain.com/templates/template.yml
+
+# Install multiple templates from URLs
+influx pkg \
+  -u https://mydomain.com/templates/template1.yml \
+  -u https://mydomain.com/templates/template2.yml
+```
+
+### Install templates from both files and URLs
+To install templates from both files and URLs in a single command, include multiple
+file or directory paths and URLs, each with the appropriate `-f` or `-u` flag.
+
+```sh
+# Syntax
+influx pkg -u <template-url> -f <template-path>
+
+# Example
+influx pkg \
+  -u https://mydomain.com/templates/template1.yml \
+  -u https://mydomain.com/templates/template2.yml \
+  -f ~/templates/custom-template.yml \
+  -f ~/templates/iot/home/ \
+  --recurse
+```
+
+### Include secrets when installing a template
+Some templates use [secrets](/v2.0/security/secrets/) in queries.
+Secret values are not included in templates.
+To define secret values when installing a template, include the `--secret` flag
+with the secret key-value pair.
+
+```sh
+# Syntax
+influx pkg -f <template-file-path> --secret=<secret-key>=<secret-value>
+
+# Examples
+# Define a single secret when installing a template
+influx pkg -f /path/to/template.yml \
+  --secret=FOO=BAR
+
+# Define multiple secrets when installing a template
+influx pkg -f /path/to/template.yml \
+  --secret=FOO=bar \
+  --secret=BAZ=quz
+```
+
+_To add a secret after installing a template, see [Add secrets](/v2.0/security/secrets/manage-secrets/add/)._
+
+## Use InfluxDB community templates
+The [InfluxDB Community Templates repository](https://github.com/influxdata/community-templates/)
+is home to a growing number of InfluxDB templates developed and maintained by
+others in the InfluxData community.
+Install community templates directly from GitHub using a template's raw code URL
```

Comment:
the template's remote URL? --how we're wording it on the github repo, so either way, we'll want to keep consistent terminology